examples/shared/commandline.c

```diff
@@ -487,3 +487,10 @@ void print_status(FILE * stream,
 {
     fprintf(stream, "%d.%02d (%s)", (status&0xE0)>>5, status&0x1F, prv_status_to_string(status));
 }
+
+int validate_coap_block_size(uint16_t coap_block_size_arg)
+{
+	return coap_block_size_arg >= 16 && \
+		coap_block_size_arg <= 1024 && \
+		coap_block_size_arg % 2 == 0;
```

Comment:
This would allow any even block size from 1 to 1024, not just 16, 32, 64, 128, 256, 512, 1024.

Reply:
I will update this part and move into `core/utils.c`